### PR TITLE
Samantha/maze2

### PR DESCRIPTION
**Pull request by zaksaja**
[Link to check-in or debrief if applicable](link to google doc goes here)

<h2>Summary</h2>
 
This PR is a project in its preliminary working stages. Through utilizing placeholders before diving into heavy detail work, it is still in it's very rough stage as I was deliberating exactly what kind of game I wanted to make. My intention is to eventually create a maze where players have to search for 'light' in the dark. My goal is to give the end game a dungeon-feel and make some pixel art assets for the final look.  

<h2>Details</h2>

The game itself is mainly just placeholders as of current but one detail is the darkness that players are surrounded by and must navigate through. I plan to use this element to make the gameplay more entertaining as players attempt to find their way through the maze of trap doors and winding corridors. I will be adding a start scene and end scene later on as well once you reach the exit of the map and find the unlit lantern to illuminate the darkness of the maze. 

<h2>How to Test</h2>

- How do we play this?
Mainly moving around the map and interacting with the space currently. (WASD)
<h2>How to Review</h2>

I feel like I would like to mainly receive feedback in terms of inspiring an objective in the final game. Currently my goal is to illuminate the dark of the space to 'win' but I believe maybe collecting power up items along the way could possibly expand your view? My primary questions are; Ideas for objectives or items to place within the map? Maybe some aesthetic inspiration for the overall look?

### CODE REVIEW

File: Assets/samantha zak/Scripts/PlayerControl1.cs [hunk 1]
```diff
@@ -0,0 +1,24 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PlayerControl : MonoBehaviour
```
Comment (jasonzli) [inline Assets/samantha zak/Scripts/PlayerControl1.cs:5]: namespace, but this class is empty? Placeholder?

File: Assets/samantha zak/Scripts/PlayerMove.cs [hunk 2]
```diff
@@ -0,0 +1,21 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PlayerMove : MonoBehaviour
```
Comment (jasonzli) [inline Assets/samantha zak/Scripts/PlayerMove.cs:5]: Also an empty class? Is this a placeholder?

File: Assets/samantha zak/Scripts/PlayerMovement.cs [hunk 3]
```diff
@@ -0,0 +1,33 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+
+public class PlayerMovement : MonoBehaviour
```
Comment (jasonzli) [inline Assets/samantha zak/Scripts/PlayerMovement.cs:6]: Namespace

**Review by jasonzli — CHANGES_REQUESTED** [review]
Looks like we're in the beginning stages of this project.

Namespaces are missing, but the big thing that is problematic is all the URP, project changes and miscellaneous .meta files that shouldn't be in here.

Reorganize your project and also delete the extra files that you don't need.

Need to fill out the PR description!

**Review by pecom291 — APPROVED** [review]
This is actually really cool though I will admit I could not find the exit ):. I think the game play is great and the idea to make the spinning objects have collision and move with you is a really cool game aspect. Adding more walls and obstacles to it will definitely help as the obstacles kind of give an idea of where players can go (or fool them). Overall I think it's coming along great and I'd happily play it when its updated again!

**Review by redmoe — CHANGES_REQUESTED** [review]
Game is looking good!
- [x] Move assets into a folder into a separate game folder inside of your named folder ( Assets/samantha zak -> Assets/samantha zak/Arcade)
- [ ] Add namespaces to scripts (see Jason's comments)
- [ ] Move assets outside of your game's folder into your game's folder (ObstacleAssetStars, Bouncy, hinge1)

**Pull request merged**